### ISSUE TITLE
[android] Screen dims & locks while video is playing

### ISSUE THREAD

**Issue by tostr7191** [TheWidlarzGroup/react-native-video#744]
Hi

On iOS this works correctly, but on android I get this behaviour. After the inactivity timeout the screen goes dim, then black (phone locks). I have found nothing in the docs about this, no settings or something. Is this a bug, or am I just missing something? I have tried it with:

```
"react-native-video": "^1.0.0",
```
and
```
"react-native-video": "^1.2.0",
```

Same behaviour in both cases.

Best,
tostrio

**Comment by tostr7191** [comment]
There is a similiar issue here: #687 
No responses though.

**Comment by shooterRao** [comment]
I have also encountered the same problem..Ios works correctly but android can't work...

**Comment by shooterRao** [comment]
@tostrio Hi,I have just solved this problem,use 'react-native-keep-awake', I use it in componentWillMount(){ KeepAwake.activate(); } and componentWillUnmount(){ KeepAwake.deactivate(); },
this problem is solved！

**Comment by tostr7191** [comment]
@shooterRao Hi, thanks for the heads up! 

We have opted to use `react-native-keep-awake` as well. But I still think this should be fixed in the video player. Preventing screen timeout seems to be something that the player should include, and not something that should need an external dependency (at least from my point of view). Not to mention that the behaviour is not consistent across platforms.

**Comment by shooterRao** [comment]
@tostrio yeah,I agree with you on this point,I also hope this bug can be solved.

**Comment by cobarx** [comment]
Fixed by #1117. Will be included in 3.2.0
